invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/RDMRecordResultsGridItem.js

```diff
@@ -0,0 +1,31 @@
+// This file is part of InvenioRDM
```

Comment:
Does the filename match the name which would be used by a user in ``my-site/assets/templates/search/<filename>``?

Reply:
Nope. Fixed now :)